application/controllers/collection_instrument.py

```diff
@@ -155,7 +155,7 @@ def upload_to_bucket(self, exercise_id, file, ru_ref=None, classifiers=None, ses
             seft_ci_bucket.upload_file_to_bucket(file=file)
             instrument.seft_file.gcs = True
         except Exception:
-            log.exception("An error occurred when trying to put SEFT CI in bucket")
+            log.error("An error occurred when trying to put SEFT CI in bucket")
```

Comment:
I think you miss understood my comment, this should stay as exception but also raise the error. I.e. 

```
  except Exception as e:
            log.exception("An error occurred when trying to put SEFT CI in bucket")
            raise e
```